Source/Extensions/Blazorise.DataGrid/Enums/DataGridEditMode.cs

```diff
@@ -19,4 +19,10 @@ public enum DataGridEditMode
     /// Values will be edited in the modal dialog.
     /// </summary>
     Popup,
+
+    /// <summary>
+    /// Any cell value can be edited directly allowing for rapid editing of data.
+    /// <para>It is of note that certain features may be disabled. e.g edit command column, etc...</para>
+    /// </summary>
+    Cell
```

Comment:
I guess `Cell` is an OK name in this case. But I'm already thinking about what will we name a batched cell edit mode?  🤔

Maybe BatchedCell?

Reply:
Despite batch mode being most used with cell type based edit (i.e excel like edit).

I was thinking that it could be a flag named `Batch` or `BatchEdit` that works with all edit modes, that should give some flexibility.

Instead of saving the data right away it will just keep in memory until **save or cancel** is hit. Whether I edit using modal, form, etc... the cells that were edited will have a visual indicator so the user can keep track of what's edited.

Reply:
That makes sense. In fact I already see the benefit od doing it that way.

Reply:
Right, yea, I think the flexibility will be a really good aspect of it.

I'm not sure if we should track edited values across page navigation, but I think we should. What do you think?

Reply:
> I'm not sure if we should track edited values across page navigation, but I think we should. What do you think?

We should track them across pages.